### ISSUE TITLE
Incorrect callsite+callstack for failed assertion in a method of some class

### ISSUE THREAD

**Issue by AlexanderMoskovkin** [DevExpress/testcafe#1267]
### Are you requesting a feature or reporting a bug?

bug

### What is the current behavior?

```sh
      AssertionError: expected false to be truthy

      Browser: Chrome 56.0.2924 / Windows 10 0.0.0

          6 |    }
          7 |}
          8 |
          9 |async function fn (t) {
         10 |    // any code
       > 11 |}
         12 |
         13 |test('test', async t => {
         14 |    const page = new Page();
         15 |
         16 |    await page.expect(t);

         at <anonymous> (c:\temp\example\test.js:11:276)
         at Page.expect (c:\temp\example\test.js:11:283)
         at <anonymous> (c:\temp\example\test.js:16:20)
         at test (c:\temp\example\test.js:17:44)
```

### What is the expected behavior?

It should be an error on the line 5.

#### Provide the test code and the tested page URL (if applicable)

Here is a simple test to reproduce the problem:

```js
fixture `f`;

class Page {
    async expect (t) {
        await t.expect(false).ok();
    }
}

async function fn (t) {
    // any code
}

test('test', async t => {
    const page = new Page();

    await page.expect(t);
});
```

### Specify your

* operating system: Win 10
* testcafe version: 0.14.0-alpha1
* node.js version: v6.9.1

**Comment by inikulin** [comment]
@AlexanderMoskovkin Is it regression after https://github.com/DevExpress/testcafe/issues/1226?

**Comment by AlexanderMoskovkin** [comment]
No, it's reproduced with the 0.13.0 version too. (#1226 fix added in 0.14.0-alpha1)

**Comment by georgiy-abbasov** [comment]
keyword `async` brokes lines in transpilled test file
Before
https://gist.github.com/georgiy-abbasov/7955e6d3494168afe150a70aa8da0728
After
https://gist.github.com/georgiy-abbasov/c679f3a512123d15723f099302d21da2

As workaround it's possible to use functional expression instead of function declaration.
Before
https://gist.github.com/georgiy-abbasov/82f9b8a4660d8b1484b8e0bf76032f02
After
https://gist.github.com/georgiy-abbasov/1b931e5b36ae857f471cdb3d869dcdd0

It's still investigating
